### PR TITLE
move config to nl-ohif, get DICOM data from GCP

### PR DESCRIPTION
**Pull request by m00n620**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/viewer/public/config/newlantern.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      namespace: 'org.ohif.default.dataSourcesModule.dicomweb',
+      sourceName: 'dicomweb',
+      configuration: {
+        name: 'New Lantern',
+        wadoUriRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
+        qidoRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
+        wadoRoot:
+          'https://healthcare.googleapis.com/v1/projects/new-lantern/locations/us-central1/datasets/development/dicomStores/wen-dev/dicomWeb',
```
Comment (shivasuri) [inline platform/viewer/public/config/newlantern.js:18]: Needs to be set during runtime, maybe based on environment variable?

File: platform/viewer/src/state/accessToken.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { useState, createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+const accessTokenContext = createContext(null);
+const { Provider } = accessTokenContext;
+
+export const useAccessToken = () => useContext(accessTokenContext);
+
+export function AccessTokenProvider({ children, value: token }) {
```
Comment (shivasuri) [inline platform/viewer/src/state/accessToken.js:9]: what is `children`?

File: platform/viewer/public/config/newlantern.js (hunk 1 above)
Comment (danihodovic) [reply platform/viewer/public/config/newlantern.js:18]: The urls are also set on an instance level. Maybe we could retrieve one instance and get the base url. 

https://app.oldlantern.dev/admin/core/instance/

File: platform/viewer/public/config/newlantern.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      label: 'W/L Preset 8',
+      keys: ['8'],
+    },
+    {
+      commandName: 'windowLevelPreset9',
+      label: 'W/L Preset 9',
+      keys: ['9'],
+    },
+  ],
+};
```
Comment (danihodovic) [inline platform/viewer/public/config/newlantern.js:127]: If hotkeys are the defaults, why do we need to modify them?

https://github.com/OHIF/Viewers/blob/8f2604509d24092b721f1333289d838820d68cac/platform/viewer/public/config/default.js#L34

File: platform/viewer/public/config/newlantern.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
+const config = {
```
[the remark this answers is not in the document]
Comment (danihodovic) [reply platform/viewer/public/config/newlantern.js:1]: This is a good idea.

File: extensions/default/src/Toolbar/ToolbarLayoutSelector.jsx [hunk 5]
```diff
@@ -9,18 +9,18 @@ function LayoutSelector() {
   const [isOpen, setIsOpen] = useState(false);
   const [viewportGridState, viewportGridService] = useViewportGrid();
```
Comment (shivasuri) [inline extensions/default/src/Toolbar/ToolbarLayoutSelector.jsx:11]: Can we add a comment on why we're commenting these lines out?

File: extensions/default/src/ViewerLayout/index.jsx [hunk 6]
```diff
@@ -5,14 +5,13 @@ import {
...
   UserPreferences,
-  AboutModal,
   Header,
   useModal,
 } from '@ohif/ui';
 
 import i18n from '@ohif/i18n';
 import { hotkeys } from '@ohif/core';
-import { useNavigate } from 'react-router-dom';
+// import { useNavigate } from 'react-router-dom';
```
Comment (shivasuri) [inline extensions/default/src/ViewerLayout/index.jsx:14]: can we delete?

File: platform/viewer/src/routes/Mode/Mode.jsx [hunk 7]
```diff
@@ -139,26 +167,26 @@ export default function ModeRoute({
...
+  //       query,
+  //     });
+  //     setStudyInstanceUIDs(studyInstanceUIDs);
+  //   };
+
+  //   initializeDataSource(params, query);
+  //   return () => {
+  //     layoutTemplateData.current = null;
+  //   };
+  // }, [location]);
```
Comment (shivasuri) [inline platform/viewer/src/routes/Mode/Mode.jsx:184]: delete?

**Pull request merged**